### PR TITLE
Allow isomorphic use

### PR DESCRIPTION
**Pull request by corbifex**
#8

### CODE REVIEW

File: src/howler.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+var Howler
```
Comment (thangngoc89) [inline src/howler.js]: Could you use let ?

File: src/howler.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+if (typeof window !== 'undefined') {
+  Howler = require('howler')
+}
+
+module.exports = Howler
```
Comment (thangngoc89) [inline src/howler.js]: Could you use ES6 export syntax ?

**Comment by thangngoc89** [comment]
Looks great. :+1: 

Please squash those commit into one and I'll will merge asap

File: src/howler.js (hunk 2 above)
Comment (corbifex) [inline src/howler.js]: No then you get always undefined, I think because the ES6 export will fire before the if statement.

Comment (thangngoc89) [inline src/howler.js]: Alright, just live it there. I have a hard time dealing with es6 and requirejs import/export.

Comment (corbifex) [inline src/howler.js]: This file would not be needed if Howler does error handling for the navigator error on server side. But normally howler is not requested on server side I suppose.

Comment (thangngoc89) [inline src/howler.js]: Howler was never meant to use in server :) .

**Comment by thangngoc89** [comment]
@corbifex could you squash those commits ?

**Comment by thangngoc89** [comment]
Thanks. Published this as 3.0.0

**Pull request merged**